platform/broadcom/sonic-platform-modules-accton/as7712-32x/cfg/th-as7712-32x100G.config.bcm

```diff
@@ -0,0 +1,295 @@
+# accton_as7710_32x 32x100G SDK config
```

Comment:
this needs to go to saibcm debian package, and pass the attribute validation check. You can find the instructions for the validation check in the package.